scipy/linalg/lapack.py

```diff
@@ -708,6 +708,16 @@
    cunmrz_lwork
    zunmrz_lwork
 
+   cuncsd
```

Comment:
I hope you meant within the csd methods - I did that, let me know if I missed something - it seems there is no global alphabetic order.

Reply:
alphabetical in the second letter order (the flavors are common anyways)

Reply:
Can I bother you for this part again? The reason why you placed here is because of some previous mistake we had. If you look at line 699, the order is broken from there (I should have caught this before, sorry about that). It would be great if you can fix the ordering for `geqrt` `gemqrt` `tpqrt` and `tpmqrt`.